scripts/sync_ros_packages.py

```diff
@@ -102,6 +102,9 @@
             ubuntu_distro,
             'main',
             package_architecture)
+        if arch == 'source':
+            print("Not computing diff for source architecture.", file=sys.stderr)
+            continue
```

Comment:
Why not move this block to the beginning of the for loop?

Reply:
Had to check that nothing in between the diff computation and the start of the block had any desired side-effects. I don't see that they do and so cfe66c8 moves the early continue...earlier.